verifier/check_known_issues.py

```diff
@@ -119,35 +107,44 @@ def numerals_replaced(expected, actual):
         return None
 
 
-def check_datetime_test(test_data, known_issues_list):
+def check_datetime_known_issues(test_data):
```

Comment:
rename all occurrences of `test_data` to `test` in order to be consistent with the other functions & their local variables.